### PR TITLE
Update get property testing

### LINKED ISSUE #22: [Chore] Finalize tests for the get method
The `get` method returns elements from the map if they are there, shouldn't return stuff if they have expired.

### PR DESCRIPTION
**Pull request by cmccormack**
**What is the purpose of this pull request? (put an "X" next to item)**

* [ ] Documentation update
* [x] Bug fix
* [ ] New feature
* [x] Other, please explain:

**What changes did you make? (Give an overview)**
* Added tests for `get` method
* Changed Map types to any,any to allow for symbol/object keys

**Which issue (if any) does this pull request address?**

Closes #22 

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

**Comment by cmccormack** [comment]
Should we define a type/interface that includes only the built-in JS types to use for the type arguments, rather than passing `any`?

File: tests/es6-timed-map.spec.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Es6TimedMap from '../src/es6-timed-map';
 
 describe('Es6TimedMap', () => {
-  let timedMap: Es6TimedMap<string, string>;
+  let timedMap: Es6TimedMap<any, any>;
```
Comment (bradtaniguchi) [inline tests/es6-timed-map.spec.ts:4]: If your asking about this, I set the 2 definitions for the tests as strings for simplicity sake, but `any` isn't supported due to the linter, so keeping it as a string should be fine **or** we can turn off the `any` check so we can do whatever during these tests

Comment (cmccormack) [reply tests/es6-timed-map.spec.ts:4]: If we change it to string, not sure how we would test for Symbol/Object keys.
> A Map's keys can be any value (including functions, objects, or any primitive).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Comment (bradtaniguchi) [reply tests/es6-timed-map.spec.ts:4]: we could just create a new `timedMap` in those tests with a different type-def, I had one main one just for simplicity stake for **most** tests.

**Review by bradtaniguchi — APPROVED** [review]
LGTM :+1: 

Thank you for your contributions!

**Pull request merged**